infrastructure/README.md

```diff
@@ -1,3 +1,74 @@
 # Infrastructure
 
 Tools and packages that are used to run a deployment of Taskcluster.
+
+## Terraform
+
+A simple module to set up some requirements for a Taskcluster deployment.
+
+To use this module, you must have configured the following terraform providers
+in your module:
+
+- aws
+- azurerm
+- rabbitmq
+
+### Requirements not managed here
+
+- A kubernetes cluster
+- Any objects in a kubernetes cluster (see K8s section below)
+- Any secrets (e.g. AWS access tokens)
+- A rabbitmq cluster with the RabbitMQ management plugin enabled
+- An SES email address set up in AWS. This cannot be created automatically by Terraform.
+
+### Usage
+
+First set the `RABBITMQ_USERNAME` and `RABBITMQ_PASSWORD` environment variables.
+
+Next, run `bin/create-rabbitmq-users.sh` to do just that. This will create the users and write their credentials to disk for you to load into k8s later.
+
+Next include this module in your terraform.
+
+```hcl
+module "taskcluster" {
+  source                    = "github.com/taskcluster/taskcluster-terraform"
+  prefix                    = "tc"
+  azure_region              = "${var.azure_region}"
+  rabbitmq_hostname         = "${var.rabbitmq_hostname}"
+  rabbitmq_vhost            = "${var.rabbitmq_vhost}"
+}
+```
+
+Then, authenticate to AWS and Azure before applying. How to do this will vary depending on what methods you use to store their credentials (e.g. `aws-vault`). For RabbitMQ, 
+
+Finally, run `create-aws-access-keys.sh` and `create-random-secrets.sh`. This will perform those operatiosn and write the output to disk  for you to load into k8s later.
+
+## K8S
+
+### Using
+
+Before using this, you need a python 3 installation with a couple packages installed via ` pip install -r requirements.txt`
+
+To render helm for all services, run `./bin/helmit.py`. To limit it to a specific service, use the `--service` flag followed by the service name. Output goes in the `charts` directory.
```

Comment:
At some point, this doc should mention what "render helm" means in slightly more detail, since that's a concept local to this repo, and I think "rendering/templating helm" refers to sticking values into extant helm charts in the rest of the world.  That could go here, or down in the `### Explanation` if it fits better there. Perhaps something like "to combine service descriptions from `services/` with their standard boilerplate from `templates` into usable helm charts, run `helmit.py`" ?

Reply:
I'll tweak the wording a bit now, and this will also be easier to explain once we generate a proper chart.